[Bug] Can't install kustomize-v3.8.2 by `make kustomize`

**Describe the bug**
<!--
A clear and concise description of what the bug is.
--> 
can't install kustomize-v3.8.2 by `make kustomize`
``` shell
❯ make manifests
installing kustomize-v3.8.2 into /Users/yuyicai/go/bin
Install succeed
go generate ./pkg/... ./apis/...
# TODO(yangsoon): kustomize will merge all CRD into a whole file, it may not work if we want patch more than one CRD in this way
/Users/yuyicai/go/bin/kustomize build config/crd -o config/crd/base/core.oam.dev_applications.yaml
make: /Users/yuyicai/go/bin/kustomize: No such file or directory
make: *** [manifests] Error 1
❯
❯ ll /Users/yuyicai/go/bin/kustomize
ls: /Users/yuyicai/go/bin/kustomize: No such file or directory
❯
❯ make kustomize
installing kustomize-v3.8.2 into /Users/yuyicai/go/bin
Install succeed
❯
❯ ll /Users/yuyicai/go/bin/kustomize
ls: /Users/yuyicai/go/bin/kustomize: No such file or directory
```

**To Reproduce**
<!--
Steps to reproduce the behavior:
1. The YAML files of Component/Trait I used.
2. The YAML file of Application I applied.
3. Other operations I have done.
4. See error
-->

**Expected behavior**
<!--
A clear and concise description of what you expected to happen.
-->

**Screenshots**
<!--
If applicable, add screenshots to help explain your problem.
-->
![image](https://user-images.githubusercontent.com/13033733/138258558-c59394d8-f674-4d68-8b7c-a8e9704efde3.png)



**KubeVela Version**

<!--
Describe your KubeVela controller or CLI version information.
-->

**Cluster information**
<!--
Describe your kubernetes cluster information.
- Kubernetes Version [e.g. 1.16.9]
-->

**Additional context**
<!--
Add any other context about the problem here.
-->

--- a/Makefile
+++ b/Makefile
@@ -332,9 +332,9 @@ KUSTOMIZE_VERSION ?= 3.8.2
 kustomize:
 ifeq (, $(shell kustomize version | grep $(KUSTOMIZE_VERSION)))
 	@{ \
-	set -e ;\
+	set -eo pipefail ;\
 	echo 'installing kustomize-v$(KUSTOMIZE_VERSION) into $(GOBIN)' ;\
-	curl -s https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh | bash -s $(KUSTOMIZE_VERSION) $(GOBIN);\
+	curl -sS https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh | bash -s $(KUSTOMIZE_VERSION) $(GOBIN);\
 	echo 'Install succeed' ;\
 	}
 KUSTOMIZE=$(GOBIN)/kustomize

